src/games/erdos_renyi.c

```diff
@@ -244,6 +244,120 @@ igraph_error_t igraph_erdos_renyi_game_gnm(
     return IGRAPH_SUCCESS;
 }
 
+igraph_error_t igraph_erdos_renyi_game_gnm_multi(
+    igraph_t *graph, igraph_integer_t n, igraph_integer_t m,
+    igraph_bool_t directed, igraph_bool_t loops, igraph_bool_t multi
+) {
+
+    igraph_integer_t no_of_nodes = n;
+    igraph_integer_t no_of_edges = m;
+    igraph_real_t no_of_nodes_real = (igraph_real_t) no_of_nodes;   /* for divisions below */
+    igraph_vector_int_t edges = IGRAPH_VECTOR_NULL;
```

Comment:
Please do not initialize data structures to values that will never be used. I know that sometimes it's taught to always initialize everything, but doing so tends to inhibit static analysers:

 - Either they complain about setting a value that is never used.
 - Or they fail to detect an effectively uninitialized access later (if it is present).

Overall, unnecessary initialization tends to hide errors in the code.